PowerEditor/src/Notepad_plus.cpp

```diff
@@ -1534,6 +1534,13 @@ void Notepad_plus::doTrim(trimOp whichPart)
 	}
 }
 
+void Notepad_plus::eol2ws()
+{
+	bool isEntireDoc = _pEditView->execute(SCI_GETANCHOR) == _pEditView->execute(SCI_GETCURRENTPOS);

```

Comment:
Just curious: is there a reason not to use the previous common code?
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/31a392ae59e1dc3494c4193e7a3df3e4e34f3a5e/PowerEditor/src/Notepad_plus.cpp#L1561

A new function `getMainSelectionLenght()` could be used in most cases.